client/src/encryption.rs

```diff
@@ -69,6 +70,28 @@ pub(crate) mod test {
         InMemSignalProtocolStore::new(p, reg).unwrap()
     }
 
+    pub fn signal_bundle_to_our_bundle(bundle: PreKeyBundle) -> KeyBundleContent {
```

Comment:
Maybe implement `impl From<PreKeyBundle> for KeyBundleContent` instead?

Reply:
cba når det er en test